### ISSUE TITLE
Path seperator ':' in windows

### ISSUE THREAD

**Issue by karellodewijk** [saadq/node-latex#15]
Windows doesn't like the : path separator very much, resulting in it not using the inputs directories. replace it with a ; for windows.

```
    const joinPaths = inputs => {
      var separator = (process.platform.indexOf('win') != -1) ? ";" : ":"
      return Array.isArray(inputs)
        ? `${inputs.join(separator)}` + separator
        : `${inputs}` + seperator
    }
```

**Comment by saadq** [comment]
Ahh, my mistake. Thanks so much for the fix, would you like to send a PR for it? If so, would you mind restyling it a bit like so:

```js
const joinPaths = inputs => {
  const separator = process.platform.includes('win') ? ';' : ':'

  return Array.isArray(inputs)
    ? inputs.join(separator) + separator
    : inputs + separator
}
```

**Comment by saadq** [comment]
Hey @karellodewijk, just wanted to check if you were still interested in writing a PR to fix this bug?

**Comment by stixx200** [comment]
There's another nice way to handle path delimiters in node: path.delimiter returns : on unix systems and ; on windows.
Check https://nodejs.org/api/path.html#path_path_delimiter

**Comment by georgyfarniev** [comment]
Hello, any progress about this issue?

**Comment by saadq** [comment]
This was a very simple fix, I think I just left it open for Hacktoberfest so someone could get their first PR, though I forgot to label the issue as `Hacktoberfest`. I'll fix this tonight, thanks for the reminder.

**Comment by saadq** [comment]
This is fixed as of v2.4.7. Please feel free to reopen this issue if any issue persists.
